docs/pages/plugins-manifest.md

```diff
@@ -106,11 +106,37 @@ function selectAll() {
+| `shortcut`    | Provides a default keyboard shortcut for the command, e.g. `ctrl shift t`. See table below for a reference of possible values.                                                                                                                                                                                                                                                                                                                                                                                    |
 | `script`      | Specifies the relative path within the plugin bundle's `Sketch` folder to the script implementing the command.                                                                                                                                                                                                                                                                                                                                                                                                    |
 | `handler`     | Specifies the name of function to be called with the command. The function should be declared at the top level of the script, and accepts a single `context` parameter which contains information such as the current document and selection. If this field is omitted the plugin will default to using a handler named `onRun`. <br/><br/>Alternatively, if you're using the `skpm` tool you may `export` the function instead of declaring it, or omit this field and declare the handler with `export default` |
 | `handlers`    | If more fine-grain control is required use the `handlers` field instead of `handler`, see below for in-depth documentation                                                                                                                                                                                                                                                                                                                                                                                        |
 
+##### Shortcut Values
+
+When setting the shortcut for a command, you can use any combination of the following keys, plus any regular character. Keep in mind that some characters may require users to press modifier keys on some international keyboard layouts. For example, to get `/` on a US keyboard you just need to press one key, but on a Spanish keyboard you need to press `shift 7` to get the same character. So a command with a `ctrl shift /` shortcut can't be triggered on a Spanish keyboard, because it gets interpreted as `ctrl /`.
```

Comment:
I'd suggest to make it more concise and move the edge case explanation to a separate paragraph and start with the recommendation first before backing it up with an example, i.e. _"test with international keyboard layouts, here's why"_:

> Command shortcuts can set using any combination of the following keys, plus any regular character.
> Make sure to test your shortcuts with different international keyboard layouts. Certain characters can require users to press modifier keys. For example a shortcut like `shift /` would **not** work with a Spanish keyboard layout. Whereas a forward slash `/` has a dedicated key on a US keyboard, it requires pressing `shift 7` on a Spanish keyboard.

Reply:
Much better, yes. Will change it 🙌